python/mxnet/initializer.py

```diff
@@ -264,7 +264,22 @@ def __call__(self, name, arr):
 
 @register
 class Zero(Initializer):
-    """Initialize the weight to 0."""
+    """Initializes weights to zero. Note that this does not initialize the biases.
```

Comment:
Yes, but that is not done by this initializer, correct? All the initializers except mixed seem to initialize only the weights.